content/cybersecurity/concepts/malware/terms/spyware/spyware.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Spyware' 
+Description: 'Spyware is malicious software that enters a person's computer, takes their data, and sends it to other third party apps or companies without the user's permission.' 
```

Comment:
The current description is giving an error in YAML due to the apostrophes being picked up as closing single quotation marks. I provided a possible edit for this below:
```suggestion
Description: 'Spyware is malicious software designed to gather personal data from a device in order to send it to other third-party apps or companies without consent from the user.' 
```